config/environments/development.rb

```diff
@@ -1,6 +1,7 @@
 Rails.application.configure do
   # Verifies that versions and hashed value of the package contents in the project's package.json
-  config.webpacker.check_yarn_integrity = true
+  config.webpacker.check_yarn_integrity = false
```

Comment:
Why this was necessary?

Reply:
bundle install was failing and required me to change this configuration, It is because I dont have yarn setup locally.